google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataClient.java

```diff
@@ -263,6 +329,7 @@ public boolean exists(String tableId, ByteString rowKey) {
    * }
    * }</pre>
    */
+  /** @deprecated Please use {@link BigtableDataClient#existsAsync(TargetId, String)} instead. */
```

Comment:
Here and everywhere else, we also need to add the `@Deprecated` annotation, see this example: https://github.com/googleapis/java-bigtable/blob/main/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataSettings.java#L239 And it can go into the other javadoc block:

```
/** 
   * Confirms asynchronously if given row key exists or not.
   * 
   * <p> Sample code:
   * ....
   * </pre>
   * 
   * @deprecated Please use ... instead
*/
@Deprecated
public ApiFuture<Boolean> existsAsync(String... )

```